skills/csharp/tests/musicskill.Tests/PlayMusicDialogTests.cs

```diff
@@ -16,6 +22,20 @@ namespace MusicSkill.Tests
     [TestClass]
     public class PlayMusicDialogTests : SkillTestBase
     {
+        [TestInitialize]
+        public void SetupLuisService()
+        {
+            var botServices = Services.BuildServiceProvider().GetService<BotServices>();
+            botServices.CognitiveModelSets.Add("en-us", new CognitiveModelSet()
+            {
+                LuisServices = new Dictionary<string, LuisRecognizer>()
+                {
+                    { "General", GeneralTestUtil.CreateRecognizer() },
+                    { "MusicSkill", PlayMusicTestUtil.CreateRecognizer() }
```

Comment:
why it is not a shared recognizer with ControlSettingsDialogTests? Since they are the same MusicSkill lu model

Reply:
Different dialog test with different utterances, it will be messy if put them together